### PR TITLE
Move Jana to alumni

### PR DESCRIPTION
**Pull request by thaJeztah**
Thanks for everything @mrjana! Don't be a stranger :smile:

ping @mavenugo @aboch @sanimej @dave-tucker @docker/core-engine-maintainers 

560e2f30c5d4251369543a4e020bc160

### CODE REVIEW

**Comment by unclejack** [comment]
LGTM

**Comment by jradhakr** [comment]
Thanks @thaJeztah. LGTM

File: MAINTAINERS [hunk 1]
```diff
@@ -150,6 +149,16 @@
 			# maryatdocker/docker-whale back in May 2015.
 			"moxiegirl",
 
+			# Jana Radhakrishnan was part of the SocketPlane team that joined Docker.
+			# As a maintainer, he was the lead architect for the Container Network
+			# Model (CNM) implemented through libnetwork, and the "routing mesh" powering
+			# Swarm mode networking.
+			#
+			# Jana started new adventures in networking, but you can find him tweeting as @mrjana,
+			# coding on GitHub https://github.com/mrjana, and he may be hiding on the Docker Community
```
Comment (tiborvass) [inline MAINTAINERS]: Wait, he has multiple github accounts now :P

Comment (thaJeztah) [inline MAINTAINERS]: LOL, yes, I just noticed :joy:

Comment (mrjana) [inline MAINTAINERS]: I erased it without evidence :-) You did not see it

Comment (thaJeztah) [inline MAINTAINERS]: hahaha

**Comment by mrjana** [comment]
LGTM

**Review by vdemeester — APPROVED** [review]
LGTM 😅

**Comment by thaJeztah** [comment]
I'll go ahead and merge :wave:

**Pull request merged**